Bio/Data/IUPACData.py

```diff
@@ -148,39 +164,30 @@ def _make_ranges(mydict):
         d[key] = (value, value)
     return d
 
+
 # Mass data taken from PubChem
 
 
 # Average masses of monophosphate deoxy nucleotides
-unambiguous_dna_weights = {
-    "A": 331.2218,
-    "C": 307.1971,
-    "G": 347.2212,
-    "T": 322.2085
-    }
+unambiguous_dna_weights = {"A": 331.2218, "C": 307.1971, "G": 347.2212, "T": 322.2085}
 
 # Monoisotopic masses of monophospate deoxy nucleotides
 monoisotopic_unambiguous_dna_weights = {
     "A": 331.06817,
     "C": 307.056936,
     "G": 347.063084,
-    "T": 322.056602
-    }
+    "T": 322.056602,
+}
 
 unambiguous_dna_weight_ranges = _make_ranges(unambiguous_dna_weights)
 
-unambiguous_rna_weights = {
-    "A": 347.2212,
-    "C": 323.1965,
-    "G": 363.2206,
-    "U": 324.1813
-}
+unambiguous_rna_weights = {"A": 347.2212, "C": 323.1965, "G": 363.2206, "U": 324.1813}
 
 monoisotopic_unambiguous_rna_weights = {
     "A": 347.063084,
     "C": 323.051851,
     "G": 363.057999,
-    "U": 324.035867
+    "U": 324.035867,
 }
 
```

Comment:
The current style for weights is even, after black monoisotopics don't fit in one line, given that is minor I think we just leave black do the styling.

Reply:
Black added the trailing comma, which other people recommend anyway in lists and dictionaries as it reduced the diff if you later add more entries.

Reply:
See also https://github.com/PyCQA/flake8-commas/